modules/oadp-api-details.adoc

```diff
@@ -0,0 +1,231 @@
+// Module included in the following assemblies:
+//
+// * oadp-api.adoc
+
+:_content-type: REFERENCE
+[id="oadp-api-details_{context}"]
+= OADP API
+
+The following tables expose the structure of the OADP API:
+
+.DataProtectionApplicationSpec
+[options="header"]
+|===
+|Property|Type|Description
+
+|`backupLocations`
+|[] link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220722142130-a02487302a02/api/v1alpha1#BackupLocation[`BackupLocation`]
+|Defines the list of configurations to use for `BackupStorageLocations`.
+
+|`snapshotLocations`
+|[] link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220722142130-a02487302a02/api/v1alpha1#SnapshotLocation[`SnapshotLocation`]
+|Defines the list of configurations to use for `VolumeSnapshotLocations`.
+
+|`unsupportedOverrides`
+|map [ link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220722142130-a02487302a02/api/v1alpha1#UnsupportedImageKey)[UnsupportedImageKey] ]  link:https://pkg.go.dev/builtin#string[string]
+|Can be used to override the deployed dependent images for development. Options are `veleroImageFqin`, `awsPluginImageFqin`, `openshiftPluginImageFqin`, `azurePluginImageFqin`, `gcpPluginImageFqin`, `csiPluginImageFqin`, `dataMoverImageFqin`, `resticRestoreImageFqin`, `kubevirtPluginImageFqin`, and `operator-type`.
+
+|`podAnnotations`
+|map [ link:https://pkg.go.dev/builtin#string[string] ] link:https://pkg.go.dev/builtin#string[string]
+|Used to add annotations to pods deployed by Operators.
+
+|`podDnsPolicy`
+|link:https://pkg.go.dev/k8s.io/api/core/v1#DNSPolicy[`corev1.DNSPolicy`]
+|Defines the configuration of the DNS of a pod.
+
+|`podDnsConfig`
+|link:https://pkg.go.dev/k8s.io/api/core/v1#PodDNSConfig[`corev1.PodDNSConfig`]
+|Defines the DNS parameters of a pod in addition to those generated from `DNSPolicy`.
+
+|`backupImages`
+|*link:https://pkg.go.dev/builtin#bool[bool]
+|Used to specify whether or not you want to deploy a registry for enabling backup and restore of images.
+
+|`configuration`
+|*link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220722142130-a02487302a02/api/v1alpha1#ApplicationConfig[`ApplicationConfig`]
+|Used to define the data protection application's server configuration.
+
+|`features`
+|*link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220722142130-a02487302a02/api/v1alpha1#Features[`Features`]
```

Comment:
Blocked until https://github.com/openshift/oadp-operator/pull/747 is merged and ideally oadp-operator repo is v1.1.0 tagged. I would want to update this link from @v0.0.0-* to @v1.1.0

Reply:
Please replace all references of https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220722142130-a02487302a02 with https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd

which contains up to date [features property](https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#Features).